README.md

```diff
@@ -90,3 +90,7 @@ library("loupeR")
 # print extra debug information
 create_bugreport_from_seurat(seurat_obj)
 ```
+
+## Tutorials
```

Comment:
Can you move this section above the `Troubleshooting` section?  Also, can you add a link to this section in the top part at line 12?

Reply:
Done.